src/java.base/share/classes/java/lang/Float16.java

```diff
@@ -679,6 +681,7 @@ public float floatValue() {
      * @jls 5.1.2 Widening Primitive Conversion
      */
     @Override
+    @IntrinsicCandidate
     public double doubleValue() {
         return (double)floatValue();
```

Comment:
Should it be handled trough Idealization step ?

ConvHF2F + ConvF2D => ConvHF2D